packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -1,10 +1,14 @@
+import { action } from "@storybook/addon-actions";
 import type { Meta, StoryObj } from "@storybook/vue3";
+import { h } from "vue";
+import SortingDataGrid from "../examples/DataGrid/SortingDataGrid.vue";
+import SortingDataGridExampleCode from "../examples/DataGrid/SortingDataGrid.vue?raw";
 import OnyxDataGrid from "./OnyxDataGrid.vue";
 
-const meta: Meta<typeof OnyxDataGrid> = {
+const meta = {
   title: "Data/DataGrid",
   component: OnyxDataGrid,
-};
+} satisfies Meta;
```

Comment:
Whats the reason for this change? Without the `typeof OnyxDataGrid` generic, the meta is not typed that good